addons/traefik/manifests/deployment.yaml

```diff
@@ -110,6 +110,8 @@ spec:
           - "--providers.kubernetescrd.ingressclass=traefik"
           - "--providers.kubernetesingress"
           - "--providers.kubernetesingress.ingressclass=traefik"
+          - "--providers.kubernetesIngress.ingressEndpoint"
+          - "--providers.kubernetesIngress.ingressEndpoint.ip=172.19.1.100"
```

Comment:
@artur-pusztai Fixed masterip of 172.19.1.100 is not always true, in k2s user can change the masterip through config.json. Get the configured masterip through `Get-ConfiguredIPControlPlane`.

Reply:
oh, I was too fast - or you too late :-)
We need to fix this later, I agree. 
We can add this two line in Enable.ps1, so we can inject the IP address dynamically.

Reply:
@artur-pusztai No issues, you can fix it in next PR.